src/tests/Microsoft.PowerFx.Core.Tests/TexlTests.cs

```diff
@@ -3038,6 +3038,48 @@ public void TestAttachmentIf()
             TestSimpleBindingSuccess("If(true, DS, DS)", schema, symbol);
         }
 
+        [Theory]
+        [InlineData("X")]
+        [InlineData("$\"Test {X}\"")]
+        [InlineData("!X")]
+        [InlineData("Table(X)")]
+        [InlineData("X.Field1")]
+        [InlineData("X + 1")]
+        [InlineData("X + Date(2022, 10, 27)")]
+        [InlineData("X * 1")]
+        [InlineData("X < 2")]
+        [InlineData("\"The\" in X")]
+        [InlineData("123 in X")]
+        [InlineData("123 in Table(X)")]
+        [InlineData("IsBlank(X)")]
+        [InlineData("IsError(X)")]
+        [InlineData("Text(X)")]
+        [InlineData("Value(X)")]
+        [InlineData("Boolean(X)")]
+        [InlineData("Index([1,2,3], X)")]
+        public void DeferredTypeTest(string script)
```

Comment:
>DeferredTypeTest [](http://example.com/codeflow?start=20&length=16)

Can we add a comment to each of these like:

// Ensure expression binds without any errors - but issues a warning for the deferred type

or

// Ensure expression issues an error , despite the deferred type.